src/Symfony/Component/FeatureFlag/README.md

```diff
@@ -0,0 +1,22 @@
+FeatureFlag Component
+=====================
+
+The FeatureFlag component allows you to split the code execution flow by
+enabling some features depending on context.
+
+It provides a service that checks if a feature is enabled. A feature is a
+callable which returns a value compared to the expected one to determine is the
+feature is enabled (mostly a boolean but not limited to).
+
```

Comment:
perhaps at the end, also need to check with repo (doc) maintainers:
there are several new components that in their readme have some code showcase
perhaps, when this PR code API is stabilized, a code example could be valuable :)

Reply:
seems to be done, resolved !